client/dashboard/data/site.ts

```diff
@@ -112,10 +113,17 @@ export interface Site {
 }
 
 export async function fetchSite( siteIdOrSlug: number | string ): Promise< Site > {
-	return await wpcom.req.get(
-		{ path: `/sites/${ siteIdOrSlug }` },
-		{ fields: JOINED_SITE_FIELDS, options: JOINED_SITE_OPTIONS }
-	);
+	try {
+		return await wpcom.req.get(
+			{ path: `/sites/${ siteIdOrSlug }` },
+			{ fields: JOINED_SITE_FIELDS, options: JOINED_SITE_OPTIONS }
+		);
+	} catch ( error ) {
+		if ( error instanceof Error && error.message.includes( '[-32710]' ) ) {
```

Comment:
Would it be more robust to check for `error.error === 'parse_error'` instead of checking for substring?

<img width="765" height="69" alt="Image" src="https://github.com/user-attachments/assets/57a0e0fe-9e0e-484e-a6ef-b686a4789843" />

Reply:
Ohh I guess because there's more kind of errors that have the same `parse_error` category

Reply:
Hmmm, actually I think `-32710` does just mean the same thing as `parse_error`.

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Spyvrag%2Spynff.ncv%2Qreebe.cuc%3Se%3Q5o221o91%23128-og

So I'll remove the sub-string match.

It'd be nice to have a more specific error than just "parse error" tbh. What's happening here is that we're trying to call the JP remove site, the API doesn't return, and rather than interpret that as an error that the JP site wasn't available, the error that gets propagated is that the response body was empty or something. So not ideal.